src/applications/personalization/dashboard-2/tests/e2e/health-care-cerner.cypress.spec.js

```diff
@@ -0,0 +1,81 @@
+import enrollmentStatusEnrolled from '@@profile/tests/fixtures/enrollment-system/enrolled.json';
+
+import {
+  makeUserObject,
+  mockLocalStorage,
+} from '~/applications/personalization/dashboard/tests/e2e/dashboard-e2e-helpers';
+
+function mockFeatureFlags() {
```

Comment:
do you think it's worth expanding this helper to accept additional feature flags to turn on/off? https://github.com/department-of-veterans-affairs/vets-website/blob/072d76c2469c07b489c875d6ad4c5ad6a9b4f262/src/applications/personalization/dashboard-2/tests/e2e/helpers.js#L1

Reply:
Also might want to start using `cy.intercept` since `route` is doing to be deprecated soon. In this case I think the API is identical so it's an easy switch.

Reply:
I don't think this helper needs expanding. After your review, I removed the logic to also turn on `show_new_schedule_view_appointments_page` since this feature flag is always turned on as confirmed with @kelsonic. I now import the feature toggle from the helpers file you just referenced 😃